build/references/command-line-interface.md

```diff
@@ -655,3 +670,29 @@ Path to JSON file that defines aliases for Virtual Machine IDs. Defaults to `~/.
 
 The above example aliases the VM whose ID is `"tGas3T58KzdjLHhBDMnH2TvrddhqTji5iZAMZ3RXs2NLpSnhH"` to `"timestampvm"` and `"timerpc"`.
 
+### X-Chain / AVM <a id="avm-config"></a>
+
+This allows you to specify a config to be passed into the X-Chain. The default values for this config are:
+
+```javascript
+{
+  "index-transactions": false,
+  "index-allow-incomplete": false
+}
+```
+
+Default values are overridden only if explicitly specified in the config.
+
+The parameters are as follows:
+
+#### Transaction Indexing
+
+`index-transactions` \(boolean\):
+
+Enables AVM transaction indexing if set to `true`. Default value is `false`. When set to `true`, AVM transactions are indexed against the `address` and `assetID` involved. This data is available via `avm.getAddressTxs` [API](exchange-chain-x-chain-api.md#avm-get-address-txs-api).
```

Comment:
I'm not sure if this ref is going to work
```
This data is available via `avm.getAddressTxs` [API](exchange-chain-x-chain-api.md#avm-get-address-txs-api).
```
could either of you please advise? @danlaine @cgcardona

Reply:
It will